stopes/pipelines/asr_bleu/compute_asr_bleu.py

```diff
@@ -0,0 +1,60 @@
+import asyncio
+import logging
+
+from stopes.pipelines.asr_bleu.configs import AsrBleuConfig
+from stopes.pipelines.asr_bleu.utils import retrieve_asr_config
+from stopes.pipelines.asr_bleu.transcribe_audio import transcribe_audio
+from stopes.pipelines.asr_bleu.retrieve_data import retrieve_data
+
+import hydra
+import sacrebleu
+
+class AsrBleu:
+    def __init__(self, config: AsrBleuConfig):
+        self.config = config
+        self.launcher = hydra.utils.instantiate(self.config.launcher)
+        self.logger = logging.getLogger("asr_bleu")
+
+    async def run(self):
+        # 1. Retrieve ASR configuration 
+        logging.info("Setting up ASR model...")
```

Comment:
Should be `logger.info()` right?

Reply:
Thanks, resolved with https://github.com/facebookresearch/stopes/pull/52/commits/a2682411272445a780ca428074c2f3c42ecd8b2e